bgpd/bgpd.h

```diff
@@ -758,6 +762,10 @@ struct bgp_nexthop {
 #define BGP_GTSM_HOPS_DISABLED  0
 #define BGP_GTSM_HOPS_CONNECTED 1
 
+/* Advertise map */
+#define CONDITION_NON_EXIST	0
```

Comment:
since this is only bool, we should use true/false instead

Reply:
sure. will change it